Sources/CoreAardvark/Logging/Logging.swift

```diff
@@ -16,13 +16,27 @@
 
 import Foundation
 
-
+/// Logs a new messages to the default Aardvark log distributor.
+///
+/// - parameter text: The text of the log message.
+/// - parameter type: The type of log message.
+/// - parameter image: An optional image to be attached to the log message. Typically used for messages of type
+/// `.screenshot`.
```

Comment:
nit: let's line up the leading whitespace here
```suggestion
///                                  `.screenshot`.
```